packages/desktop-client/src/components/modals/SelectLinkedAccountsModal.tsx

```diff
@@ -101,14 +132,17 @@ export function SelectLinkedAccountsModal({
     account => !Object.values(chosenAccounts).includes(account.id),
   );
 
-  function onSetLinkedAccount(externalAccount, localAccountId) {
-    setChosenAccounts(accounts => {
-      const updatedAccounts = { ...accounts };
+  function onSetLinkedAccount(
+    externalAccount: NormalizedAccount,
+    localAccountId: string | undefined,
+  ) {
+    setChosenAccounts((accounts: LinkedAccountIds): LinkedAccountIds => {
+      const updatedAccounts: LinkedAccountIds = { ...accounts };
 
-      if (localAccountId) {
-        updatedAccounts[externalAccount.account_id] = localAccountId;
+      if (localAccountId !== undefined) {
```

Comment:
We use if(variable) for presence checks elsewhere in the code

```suggestion
      if (localAccountId) {
```